roadmap/implementers-guide/src/runtime/router.md

```diff
@@ -277,6 +277,9 @@ the parachain executed the message.
         1. Set `limit_used_places` to `max_places`
         1. Set `limit_message_size` to `max_message_size`
         1. Set `limit_used_bytes` to `config.hrmp_channel_max_size`
+    1. Send a downward message to `recipient` notifying about an inbound HRMP channel request.
+        - The DM is sent using `queue_downward_message`.
+        - The DM is represented by an appropriate XCM message.
```

Comment:
what is the appropriate XCM message? how is it determined?

is the language vague here to avoid replicating the entire XCM spec in the guide?

Reply:
> is the language vague here to avoid replicating the entire XCM spec in the guide?

yeah, exactly...  OTOH that was written some time ago when there were no message and now at least I can state the name of the message.

Reply:
should be updated now